client/client-multi/client-c/src/kaa/platform/kaa_client_properties.h

```diff
@@ -24,8 +24,26 @@
 #ifndef POSIX_KAA_CLIENT_PROPERIES_H_
 #define POSIX_KAA_CLIENT_PROPERIES_H_
 
+#include <kaa_error.h>
+
 typedef struct {
-        unsigned long max_update_time;
+    const char *working_directory;
 } kaa_client_props_t;
 
+/**
+ * @brief      Sets the client kaa properties struct by the user-defined struct
+ *
+ * @param[in]  self  Pointer to the user-defined structure
+ * 
+ * @return     An error code
+ */
```

Comment:
**State that user should not try to free the memory owned by the structure or modify the structure until `kaa_client_stop()` has been called.**